src/app/ubs/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs-location-dashboard.component.ts

```diff
@@ -868,6 +872,16 @@ export class UbsAdminTariffsLocationDashboardComponent implements OnInit, AfterV
     return this.languageService.getLangValue(uaValue, enValue) as string[];
   }
 
+  checkUserRegion(event: Event) {
+    let filteredRegions = [];
+    const target = event.target as HTMLInputElement;
+    this.filteredRegions.pipe(takeUntil(this.destroy)).subscribe((regions) => {
+      filteredRegions = regions;
+      filteredRegions.push(this.getLangValue('Все', 'All'));
+    });
+    this.isInputRegionExisting = filteredRegions.some((el) => el.toLowerCase().includes(target.value.toLowerCase()));
```

Comment:
move it to subscribe, otherwise you need async - await

    this.isInputRegionExisting = regions.some((el) => el.toLowerCase().includes(target.value.toLowerCase()));

and remove filteredRegions  array

Reply:
this issue fixed and additional conditions were added